lib/ansible/plugins/action/__init__.py

```diff
@@ -145,9 +145,9 @@ def _configure_module(self, module_name, module_args, task_vars=None):
                                    "run 'git submodule update --init --recursive' to correct this problem." % (module_name))
 
         # insert shared code and arguments into the module
-        (module_data, module_style, module_shebang) = modify_module(module_name, module_path, module_args, task_vars=task_vars, module_compression=self._play_context.module_compression)
+        (module_path, module_data, module_style, module_shebang) = modify_module(module_name, module_path, module_args, task_vars=task_vars, module_compression=self._play_context.module_compression)
```

Comment:
And the last place where we'd change something if modify_module() does not need to return module_path.